core/tools/blueos_startup_update/blueos_startup_update

```diff
@@ -421,6 +434,12 @@ def main() -> int:
     match = re.match(r'(?P<tag>.*)-(?P<commit_number>\d+)-(?P<commit_hash>[a-z0-9]+)', current_git_version)
     tag, commit_number, commit_hash = match['tag'], match['commit_number'], match['commit_hash']
     logger.info(f"Running BlueOS: {tag=}, {commit_number=}, {commit_hash=}")
+    global config_file
+    global cmdline_file
+    config_file = locate_config_file()
+    logger.info(f"config.txt found at {config_file}")
+    cmdline_file = locate_cmdline_file()
+    logger.info(f"cmdline.txt found at {cmdline_file}")
```

Comment:
Shouldn't we abort if the config files were not found?

Reply:
not entirely? we have some other patches that could be useful, I think...

Reply:
yeah, I mean, skip these that depend on the config files